src/components/PaginationContainer.ts

```diff
@@ -0,0 +1,150 @@
+import { Component, ReactElement, createElement } from "react";
+import { findDOMNode } from "react-dom";
+import * as dijitRegistry from "dijit/registry";
+import * as classNames from "classnames";
+import * as dojoLang from "dojo/_base/lang";
+import * as dojoConnect from "dojo/_base/connect";
+
+import { Pagination, PaginationProps } from "./Pagination";
+import { ValidateConfigs } from "./ValidateConfigs";
+import {
+    ListView,
+    PaginationContainerProps,
+    PaginationContainerState,
+    WrapperProps,
+    parseStyle
+} from "../utils/ContainerUtils";
+import "../ui/Pagination.css";
+
+export default class PaginationContainer extends Component<PaginationContainerProps, PaginationContainerState> {
+    private navigationHandler: object;
+    private listListViewHeight: number;
+
+    constructor(props: PaginationContainerProps) {
+        super(props);
+
+        this.state = {
+            findingListviewWidget: true,
+            maxPageSize: 0,
+            offSet: 1,
+            showPageButton: true
+        };
+
+        this.updateListView = this.updateListView.bind(this);
+        this.transformListView = this.transformListView.bind(this);
+
+        this.navigationHandler = dojoConnect.connect(
+            props.mxform,
+            "onNavigation",
+            this,
+            dojoLang.hitch(this, this.validateConfigs)
```

Comment:
Inline all...
`this.navigationHandler = dojoConnect.connect(props.mxform, "onNavigation", this, () => this.validateConfigs());`
OR
```
this.validateConfig = this.validateConfig.bind(this);
this.navigationHandler = dojoConnect.connect(props.mxform, "onNavigation", this, this.validateConfigs);
```

Reply:
done